Makefile

```diff
@@ -680,6 +680,9 @@ publish_sources: $(COLLECTED_SOURCES)
 	$(QUIET): $@: Begin
 	cp pkg/sources/* $(SOURCES_DIR)
 	$(LINUXKIT) $(DASH_V) pkg $(LINUXKIT_PKG_TARGET) --platforms linux/$(ZARCH) --hash-path $(CURDIR) --hash $(ROOTFS_VERSION)-$(HV) --docker $(SOURCES_DIR)
```

Comment:
In the eve Makefile rule we also have
--docker $(if $(strip $(EVE_REL)),--release) $(EVE_REL)$(if $(strip $(EVE_REL)),-$(HV))

I don't know exactly what that does; do we need the same here?

Reply:
@eriknordmark I have no idea how you get that really cool-looking formula-type formatting, but I want to know.

Reply:
I don't actually know if we need it. @yash-zededa I think you will need to run a few `make -n` with various cases to see what it does. 😢

Reply:
I have executed few make -n and few actual make by passing the EVE_REL=snapshot along with the other var and the output from eve and eve-sources was similar in terms of hash. The only thing that concerns me is on the ARCH or ZARCH for the image tag. I am planning to do few more runs but pushed this MR for the initial review. Will share the results.

Reply:
> @eriknordmark I have no idea how you get that really cool-looking formula-type formatting, but I want to know.

I guess the Makefile/ + shell syntax is interpreted as markdown in interesting ways. I haven't yet figured out the quoting scheme; let me try editing the above comment to make it more boring.